### PR TITLE
Added doorkick plugin :)

### PR DESCRIPTION
**Pull request by jopnine**
I shall improve this on the future

### CODE REVIEW

File: doorkick/sh_plugin.lua [hunk 1]
```diff
@@ -0,0 +1,89 @@
+
+PLUGIN.name = "Doorkick"
+PLUGIN.author = "B4tr1p"
+PLUGIN.description = "Adds doorkick to HL2RP."
```
Comment (jopnine) [inline doorkick/sh_plugin.lua:4]: I shall make a better desc in the future

File: doorkick/sh_plugin.lua [hunk 2]
```diff
@@ -0,0 +1,89 @@
+
+PLUGIN.name = "Doorkick"
+PLUGIN.author = "B4tr1p"
+PLUGIN.description = "Adds doorkick to HL2RP."
+
```
Comment (impulsh) [inline doorkick/sh_plugin.lua:5]: Missing a license

File: doorkick/sh_plugin.lua [hunk 3]
```diff
@@ -0,0 +1,89 @@
+
+PLUGIN.name = "Doorkick"
+PLUGIN.author = "B4tr1p"
+PLUGIN.description = "Adds doorkick to HL2RP."
+
+
```
Comment (impulsh) [inline doorkick/sh_plugin.lua:6]: `PLUGIN.schema` should be `"HL2 RP"`

File: doorkick/sh_plugin.lua [hunk 4]
```diff
@@ -0,0 +1,89 @@
+
+PLUGIN.name = "Doorkick"
+PLUGIN.author = "B4tr1p"
+PLUGIN.description = "Adds doorkick to HL2RP."
+
+
+ix.config.Add("strneeded", 0.5, "How much strength is needed to kick a dor.", nil, {
```
Comment (impulsh) [inline doorkick/sh_plugin.lua:7]: Typo: `dor` instead of `door`

File: doorkick/sh_plugin.lua [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+		-- Validate it is a door.
+		print(ix.config.Get("combineonly", false) )
+		print(client:IsCombine())
+	if(ix.config.Get("combineonly", false)) then
+		if(client:IsCombine()) then
+					if (IsValid(entity) and entity:IsDoor() and !entity:GetNetVar("disabled")) then
+						if (client:GetPos():Distance(entity:GetPos())< 100) then	
+							if (client:GetCharacter():GetAttribute("str", 0) > ix.config.Get("strneeded", 15)) then
+								if(current > 95) then		
+									print(client:GetCharacter():GetAttribute("str", 0))
```
Comment (impulsh) [inline doorkick/sh_plugin.lua:36]: This print should be removed

File: doorkick/sh_plugin.lua [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+					end
+		else
+		client:Notify("You are not a combine")
+		end
+	else
+		if (IsValid(entity) and entity:IsDoor() and !entity:GetNetVar("disabled")) then
+			if (client:GetPos():Distance(entity:GetPos())< 100) then	
+				if (client:GetCharacter():GetAttribute("str", 0) > ix.config.Get("strneeded", 15)) then
+					if(current > 95) then		
+						print(client:GetCharacter():GetAttribute("str", 0))
```
Comment (impulsh) [inline doorkick/sh_plugin.lua:65]: This print should be removed

File: doorkick/sh_plugin.lua [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+    adminOnly = true,
+
+	OnRun = function(self, client, arguments)
+
+		-- Get the door the player is looking at.
+		local entity = client:GetEyeTrace().Entity
+		local current = client:GetLocalVar("stm", 0)
+		-- Validate it is a door.
+		print(ix.config.Get("combineonly", false) )
+		print(client:IsCombine())
```
Comment (impulsh) [inline doorkick/sh_plugin.lua:29]: These prints should be removed

File: doorkick/sh_plugin.lua [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+		if (IsValid(entity) and entity:IsDoor() and !entity:GetNetVar("disabled")) then
+			if (client:GetPos():Distance(entity:GetPos())< 100) then	
+				if (client:GetCharacter():GetAttribute("str", 0) > ix.config.Get("strneeded", 15)) then
+					if(current > 95) then		
+						print(client:GetCharacter():GetAttribute("str", 0))
+						client:ConsumeStamina(110)
+					--	 client:ForceSequence("adoorkick") --Entra que nem maluco
+					--	client:ForceSequence("kickdoor")
+						client:ForceSequence("kickdoorbaton")
+						timer.Simple( 0.5, function()
```
Comment (impulsh) [inline doorkick/sh_plugin.lua:70]: `ForceSequence` callback should be used here too

**Review by impulsh — CHANGES_REQUESTED** [review]

**Pull request closed without merging**